packages/main/src/SegmentedButton.js

```diff
@@ -0,0 +1,163 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
```

Comment:
It would be great if you add keyboard navigation:
- Up/Down and Left/Right arrow keys to navigate between the buttons
- TAB/SHIFT+TAB should leave the group perhaps
You can check how it`s in openui5.